app.py

```diff
@@ -49,13 +49,14 @@ def main_prediction():
     """
     st.markdown(html_temp,unsafe_allow_html=True)
     st.markdown("""<br>""",unsafe_allow_html=True)
-    x = st.text_input("X")
-    y = st.text_input("Y")
-    month = st.text_input("Month")
-    ffmc = st.text_input("FFMC")
-    dmc = st.text_input("DMC")
-    dc = st.text_input("DC")
-    temp = st.text_input("Temp")
+    x = st.text_input("X","Enter an integer e.g. 5")
+    y = st.text_input("Y","Enter an integer e.g. 5")
+    month = st.text_input("Month","Enter a numeric representation of the month e.g. if February enter 2")
+    ffmc = st.text_input("FFMC","Enter any number b/w 18.7 to 96.20")
+    dmc = st.text_input("DMC","Enter any number b/w 1.1 to 291.3")
+    dc = st.text_input("DC","Enter any number b/w 7.9 to 860.6")
+    temp = st.text_input("Temp","Enter any number b/w  2.2 to 33.30")
+    # num = st.number_input("Enter a number",5)
```

Comment:
Dont provide ranges. Just provide an example value.

Reply:
I am not complaining but as a maintainer, you should be careful, every time I am adding any features and I was also showing that in the form of screenshots then I was asking for "should I push the changes?".every time you were saying yes ok push the changes.
So why to ask for changes now.
Obviously, I will change what you have asked for, but make sure these are the last changes.

Reply:
@soumya997 How am I supposed to know if your code is static or dynamic based on your screenshots? If you dont push your changes I can't see what code you have written.

Reply:
And the range was mentioned in the readme, Value of those parameters ranges between that and if you give more than or less than the values of range it will not make any sense, not for the model(as for the model it would be new data and made predictions would be wrong) and not in general.
So, review the above statement👆 and say you still want to see some changes or not.

Reply:
Adding a place holder is static, and you can easily see that via a screenshot.
When it's about the dynamic feature that you requested for the contributors page for that you can say that you cant see via a screenshot, and for that I have handed over to you, see comment below.

Reply:
> And the range was mentioned in the readme, Value of those parameters ranges between that and if you give more than or less than the values of range it will not make any sense,

Looks like you haven't researched about the features. The values provided in the readme were the ranges within the dataset and not the actual parameter ends for that feature. for e.g

* FFMC ranges from 0-101 and not 18.7 to 96.20
* DMC is open ended which means it can have any value and it not closed between 1.1 to 291.3
* DC ranges from 0 - 1000 and not in between 7.9 to 860.0
* ISI is also open ended.
* Same goes for temp, wind, rain etc.

> not for the model(as for the model it would be new data and made predictions would be wrong) and not in general.

Well in that case the model is overfitting and should not be used in the first place.

Reply:
> Well in that case the model is overfitting and should not be used in the first place.

Its a good point but I said that on mode retraining perspective, do you know that after deployment there is a concept called model retraining, retraining of the model happens when the model starts to perform worse every time and it performs worse because it needs to predict the data which all new to it.
And except that and I am also not complaining by saying that but the csv file was of 500 rows.

Reply:
> Looks like you haven't researched about the features. The values provided in the readme were the ranges within the dataset and not the actual parameter ends for that feature. for e.g

Thanks for pointing that out yes I did not and I am sorry for that.

Reply:
Ok @Nibba2018 , I am pushing the changes except for the contributors commit feature.

Reply:
```
x = st.text_input("X","Enter an integer e.g. 5")
y = st.text_input("Y","Enter an integer e.g. 5")
```

This is the code for variable "X" and "Y", should I change the placeholders too like: `e.g. 5`, or should I let it be?

Reply:
> And except that and I am also not complaining by saying that but the csv file was of 500 rows.
I know, we are not "actually" deploying a model so it alright. :laughing: . Mentioning such a range is like asking your teacher to ask viva questions from the only chapter that you have studied. :laughing: 

> Its a good point but I said that on mode retraining perspective, do you know that after deployment there is a concept called model retraining, retraining of the model happens when the model starts to perform worse every time and it performs worse because it needs to predict the data which all new to it.

Yeah thats when we want to retrain the model so that it can learn new features from the new data but in this case we only want it to predict.

> This is the code for variable "X" and "Y", should I change the placeholders too like: e.g. 5, or should I let it be?

This is fine, no need to change this.

>  Ok @Nibba2018 , I am pushing the changes except for the contributors commit feature.

LGTM! :rocket:

Reply:
@Nibba2018 I have pushed the changes, please check.

Reply:
> I know, we are not "actually" deploying a model so it alright. 😆 . Mentioning such a range is like asking your teacher to ask viva questions from the only chapter that you have studied. 😆


If you remember properly I did use the whole 500 data samples for training, only 80% was for training, so the rest of the data was for validation. So the whole range of each feature (like- for FFMC- 18.7 to 96.20) was not going training. 
There I just tried to mention what was provided in the readme.
And questions are always out of the syllabus.